src/tracking.tsx

```diff
@@ -46,7 +46,7 @@ export class Tracking {
   constructor({
     _isDoNotTrackEnabled = isDoNotTrackEnabled,
     _reactGA = ReactGA,
-    trackingEnabled = Boolean(process.env.REACT_APP_TRACKING_ENABLED),
+    trackingEnabled = process.env.REACT_APP_TRACKING_ENABLED === 'true',
```

Comment:
curiosity: why does `REACT_APP_TRACKING_ENABLED` change to `'true'` from `true` along the way from the `.env` file to here?

Reply:
I think it's just the nature of how `process.env` works. It always returns strings to us.